test/instrumentation/modules/@aws-sdk/client-sns.test.js

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright Elasticsearch B.V. and other contributors where applicable.
+ * Licensed under the BSD 2-Clause License; you may not use this file except in
+ * compliance with the BSD 2-Clause License.
+ */
+
+'use strict';
+
+// Test SNS instrumentation of the '@aws-sdk/client-sns' module.
+//
+// Note that this uses localstack for testing, which mimicks the SNS API but
+// isn't identical. Some known limitations:
+// TODO: check if there are limitations
```

Comment:
```suggestion
// isn't identical.
```

Can perhaps clean out the TODO if your tests are passing. :)